console/src/app/modules/form-field/field/form-field.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="cnsl-form-field-wrapper" (click)="_control.onContainerClick && _control.onContainerClick($event)">
+  <ng-content select="cnsl-label"></ng-content>
+  <div class="cnsl-rel" #inputContainer>
+    <ng-content></ng-content>
+    <ng-content select="cnslSuffix"></ng-content>
+  </div>
+  <div *ngIf="errori18nKeys$ | async; let errs">
+    <div class="cnsl-form-field-subscript-wrapper">
+      <div class="cnsl-form-field-hint-wrapper" *ngIf="errs.length === 0" [@transitionMessages]="_subscriptAnimationState">
+        <ng-content select="cnsl-hint"></ng-content>
+      </div>
+      <div class="cnsl-form-field-error-wrapper" *ngIf="errs.length > 0" [@transitionMessages]="_subscriptAnimationState">
```

Comment:
ensure that other error directives are included as well

Reply:
done